hibernate-rx-core/src/main/java/org/hibernate/rx/impl/RxSessionInternalImpl.java

```diff
@@ -189,7 +258,7 @@ private Boolean getReadOnlyFromLoadQueryInfluencers() {
 						null,
 						object,
 						isCascadeDeleteEnabled,
-						((StatefulPersistenceContext) getPersistenceContextInternal())
+						((RxPersistenceContextAdapter) getPersistenceContextInternal())
 								.isRemovingOrphanBeforeUpates(),
```

Comment:
We should promote `.isRemovingOrphanBeforeUpates()` to `public` in Hibernate core.

Reply:
I'm preparing a PR for ORM to add another change of this type. I will include this

Reply:
Oops, I already implemented it and was about to send a PR. But I guess it's better if you roll it into your PR. So I will scrap mine.